### ISSUE TITLE
Export individual "formatRow" and "formatValue" functions

### ISSUE THREAD

**Issue by rdmurphy** [d3/d3-dsv#57]
Hello!

I've found myself more and more using `d3-dsv` as a sort of a Swiss Army knife of CSV manipulation — if I want to ensure that an Array of data gets properly delimited, I'll reach for `formatRows` and trust it'll do the right thing. But when I'm working with Node.js streams and looping through data line-by-line (meaning I don't have access to the entire "body" of data at any given time), it'd be neat if I could tap directly into `formatRow`.

It's not end of the world to have to use `formatRows` instead, but it requires me to create a dummy Array to wrap the individual row of data with every pass, which feels a little dirty.

```js
const createStreamWriter = (outputPath, columns) => {
  const writer = fs.createWriteStream(outputPath)
  // with dsv.csvFormatRow, I could just pass in my single Array of values
  writer.write(`${dsv.csvFormatRows([columns])}\n`)

  return data => {
    writer.write(`${dsv.csvFormatBody([data], columns)}\n`)
  }
}
```

In the case of `formatBody`, it kind of defeats the point to "singularize" that, but if `formatValue` was made available I could reproduce the effect by creating my own single row function built on it for zipping purposes.

Thank you! Looking forward to your thoughts.

**Comment by mbostock** [comment]
I’m cool with exporting formatRow and formatValue. :+1:

**Comment by curran** [comment]
Concrete tasks appear to include:

 * [ ] Export `formatRow` https://github.com/d3/d3-dsv/blob/49726009255484987c7f2028f1cc9d80ab43f5cb/src/dsv.js#L144
 * [ ] Add documentation for `formatRow` to the README

**Comment by rdmurphy** [comment]
@curran I see now I named the pull request the wrong thing months ago but I've already done those four things here: https://github.com/d3/d3-dsv/pull/58

**Comment by curran** [comment]
Oh excellent! Somehow I didn't see that PR. Nice work!

**Comment by mbostock** [comment]
Fixed in 437da9a9ab034e4a42d4b0f5e29d833a96069a4d.
